### ISSUE TITLE
Freebsd Support

### ISSUE THREAD

**Issue by bokay** [appsignal/appsignal-elixir#114]
Hi and thank you for taking into account the Elixir community.

**The issue**
When I deploy the application on a Freebsd server with edeliver, during the compiling phase, I get the error:

```
==> appsignal
could not compile dependency :appsignal, "mix compile" failed. You can recompile this dependency with "mix deps.compile appsignal", update it with "mix deps.update appsignal" or clean it with "mix deps.clean appsignal"
** (FunctionClauseError) no function clause matching in Mix.Appsignal.Helper.map_arch/1
    mix_helpers.exs:103: Mix.Appsignal.Helper.map_arch('amd64-portbld-freebsd11.0')
    mix_helpers.exs:13: Mix.Appsignal.Helper.ensure_downloaded/0
    /git/my_app/builds/deps/appsignal/mix.exs:6: Mix.Tasks.Compile.Appsignal.run/1
    (mix) lib/mix/task.ex:294: Mix.Task.run_task/3
    (elixir) lib/enum.ex:1229: Enum."-map/2-lists^map/1-0-"/2
    (mix) lib/mix/tasks/compile.all.ex:19: anonymous fn/1 in Mix.Tasks.Compile.All.run/1
    (mix) lib/mix/tasks/compile.all.ex:37: Mix.Tasks.Compile.All.with_logger_app/1
    (mix) lib/mix/task.ex:294: Mix.Task.run_task/3
```

**Expected result**
I'd expect the appsignal library compiling on Freebsd too.

**Note**
I haven't attempted to map Freebsd to Darwin as this does not seem correct.

**Comment by tombruijn** [comment]
Hi @bokay!

We don't support FreeBSD at the moment. We have an issue for it on our agent repo. It's not on our roadmap yet, but sometime soon I hope

For now I'll close the issue.

**Comment by tombruijn** [comment]
Hi @bokay,

It's finally here! We've just shipped `1.4.0-alpha.1` with 64-bit FreeBSD support!
Please try it out and report any issues you may encounter at support@appsignal.com

**Comment by bokay** [comment]
Thank you very much for taking FreeBSD into consideration!
